product_docs/docs/pgd/5/reference/functions.mdx

```diff
@@ -365,6 +365,31 @@ bdr.get_node_sub_apply_lsn(node_name name)
 -   `node_name` &mdash; the name of the node that's the source of the
     replication stream whose LSN is being retrieved.
 
+### `bdr.replicate_ddl_command`
+
+Function to replicate a DDL command to a group of nodes.
+
+#### Synopsis
+
+```sql
+bdr.replicate_ddl_command(ddlcommand text, 
+                          replication_sets text[],
+                          ddl_locking text,
+                          execute_locally bool)
+```
+
+#### Parameters
+
+- `ddlcommand` &mdash; DDL command to execute.
+- `replication_sets` &mdash; An array of replication set names to apply the `ddlcommand` to. If NULL (or the fuction is only passed the `ddlcommand`), this is set to the active PGD groups's default replication set.
+- `ddl_locking` &mdash; A string sets [`bdr.ddl_locking`](/pgd/latest/reference/pgd-settings#bdrddl_locking) while replicating. 
```

Comment:
This seems to miss the default value. Maybe indicate it will default to whatever the GUC `bdr.ddl_locking` is set *on the local system* running the `bdr.replicate_ddl_command`.

Reply:
```suggestion
- `ddl_locking` &mdash; A string that sets the [`bdr.ddl_locking`](/pgd/latest/reference/pgd-settings#bdrddl_locking) while replicating. Defaults to the GUC value for `bdr.ddl_locking` on the local system that is running the replicate_ddl_command. 
```